### ISSUE TITLE
"No Internet Advertising Column" in Snowflake Warehouse

### ISSUE THREAD

**Issue by Chetri-Janak** [reapit/foundations#11358]
Hi Team,

We would like to have the following field in the screenshot to be populated in the Datawarehouse(Analytics Schema)

**Field**: No Internet Advertising
It's currently not there in PROPERTIES table.
 
![image](https://github.com/user-attachments/assets/4ec4b6d5-88c5-44da-85c2-95e1599366a5)

Thanks!

**Comment by plittlewood-rpt** [comment]
Hi @Chetri-Janak this is available in the PROPERTIES table and is called "INTERNET_ADVERTISING" - the value is inverted to what it is in AgencyCloud. Please see [the documentation](https://reapit-analytics-schema.prod.paas.reapit.cloud/#/doc/m108v3869/reapit-analytics/subject-areas/appointments/views/properties) for more details. Thanks

**Comment by Chetri-Janak** [comment]
Hi @plittlewood-rpt

It doesn't seems like to be in the PROPERTIES table with a column called "INTERNET_ADVERTISING". 
Is it something you guys added lately because till yesterday it wasn't in the documentation nor in the table, but now in the documentation it has ?

Attached is the screenshot for your reference
![image](https://github.com/user-attachments/assets/91df044f-585d-4f7e-89cf-f7eaf6aa66ac)

Thanks!

**Comment by plittlewood-rpt** [comment]
Hi @Chetri-Janak it's likely you just need to refresh your share in the data warehouse app. That field has been in place for quite a while now so it's likely you're missing other fields too. Also, your screenshot appears to be a screenshot of Power BI (or similar) and not snowflake directly. I'd recommend logging into Snowflake directly so you are sure about where the problem is. Thanks

**Comment by Chetri-Janak** [comment]
Hi @plittlewood-rpt

I have checked in Snowflake on the PROPERTIES table, I cannot find it there. And I do not think the field has been in place, because we did our checks on multiple fronts i.e checking the documentation, table columns before raising the github ticket.

Please find the screenshot for reference from Snowflake
![image](https://github.com/user-attachments/assets/adfe0cdb-c52a-4ed5-9457-9f0351f361a6)

Thanks!

**Comment by plittlewood-rpt** [comment]
Hi @Chetri-Janak as noted above, please refresh your share in the [Data Warehouse App](https://data-warehouse-app.reapit.cloud/login) - it looks like your copy is just out of date that's all, you'll likely find you're missing a lot of columns and tables as that field has been in place for many months. Thanks

**Comment by Chetri-Janak** [comment]
Yeah thanks @plittlewood-rpt we weren't aware we gotta have to do a refresh in the Data Warehouse App, but yes now since we did we're able to see it.
Follow up question, how often would we have to do this refresh and how we will know if the data and columns aren't the latest if you know what I mean ?
Thanks!

**Comment by Chetri-Janak** [comment]
Hi, kindly following up on the below query.

Thanks!

**Comment by plittlewood-rpt** [comment]
Hi @Chetri-Janak at the moment the easiest thing to do is to regularly refresh the share to make sure your version is up to date. Breaking changes will not be introduced without prior communication. You can also keep an eye on changes to the [analytics schema documentation](https://reapit-analytics-schema.prod.paas.reapit.cloud/#/). Many thanks
